spec/system/admin/bulk_order_cancellation_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+describe '
+  As an Administrator
+  I want to be able to delete orders in bulk
+' do
+  include AdminHelper
+  include AuthenticationHelper
+  include WebHelper
+
+  context "deleting orders" do
+    let!(:o1) {
+      create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                      completed_at: Time.zone.now )
+    }
+    let!(:o2) {
+      create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                      completed_at: Time.zone.now )
+    }
+
+    before :each do
+      login_as_admin_and_visit spree.admin_orders_path
+    end
+
+    it "delets orders" do
+      # Verify that the orders have a STATE of COMPLETE
+      expect(find('tbody tr').find('td:nth-child(5)')).to have_text 'COMPLETE'
+      expect(find('tbody tr:nth-child(2)').find('td:nth-child(5)')).to have_text 'COMPLETE'
```

Comment:
This is quite brittle. It's better to select elements by text.

Reply:
I agree. It didn't occur to me, when I wrote this, that I can test this using `assert_selector`. I updated the code using it just now as per your suggestion. Have a look, and thank you for the review!